### PR TITLE
WIP: run jobs for ldap bump

### PR DESCRIPTION
**Pull request by ardaguclu**
This is a fake bump PR to see the CI jobs results in https://github.com/openshift/library-go/pull/1378

### CODE REVIEW

File: pkg/oauth/handlers/default_auth_handler.go [hunk 1]
```diff
@@ -3,15 +3,14 @@ package handlers
 import (
 	"fmt"
 	"html"
+	"k8s.io/apiserver/pkg/audit"
 	"net/http"
 	"regexp"
 	"strings"
 
-	kerrors "k8s.io/apimachinery/pkg/util/errors"
```
Comment (stlaz) [inline pkg/oauth/handlers/default_auth_handler.go:10]: keep the k8s imports separate from openshift please

File: pkg/oauth/handlers/default_auth_handler.go [hunk 2]
```diff
@@ -3,15 +3,14 @@ package handlers
 import (
 	"fmt"
 	"html"
+	"k8s.io/apiserver/pkg/audit"
```
Comment (stlaz) [inline pkg/oauth/handlers/default_auth_handler.go:6]: this does not belong here

File: pkg/cmd/oauth-server/server.go [hunk 3]
```diff
@@ -25,8 +25,7 @@ import (
 	"github.com/openshift/library-go/pkg/config/serving"
 	"github.com/openshift/oauth-server/pkg/oauthserver"
 
-	// for metrics
-	_ "github.com/openshift/library-go/pkg/controller/metrics"
```
Comment (stlaz) [inline pkg/cmd/oauth-server/server.go:29]: Can you explain this change, please?

\+ add the comment back or fix it, too

File: cmd/oauth-server/main.go [hunk 4]
```diff
@@ -22,6 +23,7 @@ import (
 
 func main() {
 	stopCh := genericapiserver.SetupSignalHandler()
+	klog.InitFlags(nil)
```
Comment (stlaz) [inline cmd/oauth-server/main.go:26]: this seems wrong (see the `logs.InitLogs()` below)

File: cmd/oauth-server/main.go [hunk 5]
```diff
@@ -3,6 +3,7 @@ package main
 import (
 	goflag "flag"
 	"fmt"
+	"k8s.io/klog/v2"
```
Comment (stlaz) [inline cmd/oauth-server/main.go:6]: wrong order

**Review by stlaz — CHANGES_REQUESTED** [review]

**Comment by ardaguclu** [comment]
@stlaz thanks for review. But this PR becomes too hacky and far from a ready PR. I just want to see the CI job results after bumping ldap. Most of the changes are not related to the bump. For example, jobs returned "unknown flag --v" error and I added `klog.InitFlags`(I'd also expect `logs.InitLogs` should work) or `controller/metrics` have been moved somewhere else, etc.

**Pull request closed without merging**